go/codepipeline-build-deploy/app/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM public.ecr.aws/nginx/nginx:perl
```

Comment:
The only thing I'd change here is that you're using nginx with Perl, which brings the container size to 68MB at least. The default alpine build (  public.ecr.aws/nginx/nginx:stable-alpine ) is 16.